wire-ios-request-strategy/Sources/Request Strategies/MLS/Actions/UpdateConversationProtocolActionHandler.swift

```diff
@@ -39,7 +39,7 @@ final class UpdateConversationProtocolActionHandler: ActionHandler<UpdateConvers
         let domain = action.qualifiedID.domain
         let conversationID = action.qualifiedID.uuid.transportString()
         let messageProtocol = action.messageProtocol.rawValue
-        let path = "/conversations/\(domain)/\(conversationID)/\(messageProtocol)"
+        let path = "/conversations/\(domain)/\(conversationID)/protocol"
```

Comment:
The endpoint is always named `/protocol` and the value is send via the http body.